ADR/0049-vsa-support.md

```diff
@@ -0,0 +1,81 @@
+# ADR-0049: Verification Summary Attestations for Release Policies
+
+Date: 2025-07-02
+
+## Status
+
+Proposed
+
+## Context
+
+We are facing several challenges with our current approach to Enterprise Contract (EC) verification
+and SDL checks:
+
+1. **Release-time EC verification performance**: Running EC at release time requires checking a
+   large number of attestations, with each attestation invoking several potentially complex Rego
+   policy checks. We have multiple examples of Konflux applications with over 100 components - to
+   check these at scale requires significant amounts of memory and compute.
+2. **Pipeline arrangement limitations**: Currently we have to run all SDL checks in the build
+   pipeline, which limits our options on how to arrange pipelines and prevents us from potentially
+   moving some checks into integration tests. See [ADR 48](https://github.com/konflux-ci/architecture/pull/229).
+3. **Redundant verification of unchanged artifacts**: Pre-merge EC checks for large snapshots often
+   include one freshly built image plus a number of images that haven't changed, yet we redo the EC
+   check on all images.
+4. **Scalability concerns**: As our applications grow in complexity (like OpenShift snapshots), the
+   current approach becomes increasingly inefficient.
+5. **Release externally-built artifacts**: Our current policies for release pipelines assume
+   artifacts were built using a sanctioned Konflux Tekton pipeline. This inhibits ingestion and
+   release of artifacts from external (but trusted) systems that do not execute builds with Tekton.
+   Ex: and artifact produced from a trusted Jenkins instance, or Java artifacts built using
+   [Project Newcastle](https://github.com/project-ncl).
+
+## Decision
+
+We will adopt Verification Summary Attestations ([VSAs](https://slsa.dev/spec/v1.1/verification_summary))
+for recording SDL policy check results. These VSAs will be used as the input to release pipeline
```

Comment:
Please explain a bit how and why VSAs will help, and introduce the term so people can uderstand the ADR without going to the link.